desktop/libs/notebook/src/notebook/connectors/hiveserver2.py

```diff
@@ -744,7 +740,7 @@ def _prepare_hql_query(self, snippet, statement, session):
 
     return hql_query(
       statement,
-      query_type=QUERY_TYPES[0],
+      query_type=QUERY_TYPES[4] if hasattr(snippet, 'dialect') and snippet['dialect'] == 'hplsql' else QUERY_TYPES[0],
```

Comment:
Can we have this in separate variable and then have it here? That way in future we can change conditions in that only (or if we need to add something in that condition).

Reply:
Yes, we can. Done